test/test.js

```diff
@@ -290,7 +290,7 @@ test('uses a custom genReqId function', function (t) {
 
   const dest = split(JSON.parse)
   let idToTest
-  function genReqId (req) {
+  function genReqId (req, res) {
```

Comment:
Please add an assertion to verify that `res` is set.

Reply:
This is done